moskito-frontend/src/components/campsites/Campsites.js

```diff
@@ -0,0 +1,29 @@
+import getCampsites from '../../services/getCampsites';
+import { useState, useEffect } from 'react';
+
+export default function Campsites() {
+    const [campsites, setCampsites] = useState([])
+
+    useEffect(() => {
+        getCampsites()
+        .then(data => setCampsites(data))
+        .catch(error => console.log(error))
```

Comment:
Current fetch status could be displayed to user (e.g. through conditional rendering).